command/cp.go

```diff
@@ -530,6 +530,10 @@ func (c Copy) Run(ctx context.Context) error {
 			if c.metadataDirective == "" {
 				// default to COPY
 				c.metadataDirective = "COPY"
+				if c.src.IsRemote() && c.dst.IsRemote() && c.contentType != "" {
+					// default to REPLACE for content type change
+					c.metadataDirective = "REPLACE"
```

Comment:
Since `COPY` and `REPLACE` are referenced in multiple places, it would be better to make them consts.

Reply:
```
const (
	metadataDirectiveCopy    = "COPY"
	metadataDirectiveReplace = "REPLACE"
)
```

Is it appropriate ?

Reply:
It looks okay, it's hard to name things :)

Reply:
done.